libs/ett/auth/feature-login/src/lib/ett-authentication-link.component.ts

```diff
@@ -45,26 +47,27 @@ const selector = 'ett-authentication-link';
   ],
   template: `
     <a *rxLet="loginUrl$ as loginUrl; rxError: loginError" [href]="loginUrl"
-      >NemID or MitID</a
+      >NemID</a
     >
     <ng-template #loginError let-error="$error">
       <p class="${selector}__error">
-        NemID and MitID login are currently unavailable. Please try again later.
+        NemID login is currently unavailable. Please try again later.
       </p>
     </ng-template>
   `,
 })
 export class EttAuthenticationLinkComponent {
-  #returnUrl = this.urlGenerator.fromCommands([ettDashboardRoutePath]);
+  #absoluteReturnUrl = this.urlGenerator.fromCommands([ettDashboardRoutePath]);
 
   loginProviderError$: Observable<unknown>;
   loginUrl$: Observable<string> = this.authOidc
-    .login(this.#returnUrl)
-    .pipe(map((response) => response.url));
+    .getLogin(this.absoluteAuthenticationAppBaseUrl, this.#absoluteReturnUrl)
+    .pipe(map((response) => response.next_url));
 
   constructor(
-    private authOidc: AuthOidcHttp,
-    private urlGenerator: AbsoluteUrlGenerator
+    private authOidc: AuthHttp,
+    private urlGenerator: AbsoluteUrlGenerator,
+    @Inject(APP_BASE_HREF) private absoluteAuthenticationAppBaseUrl: string
   ) {
     this.loginProviderError$ = this.loginUrl$.pipe(
       catchError((error: unknown) => of(error))
```

Comment:
question:

Do we know the type of error inside "catchError" above?

Reply:
No, and we don't care since we only support the happy path userflow for now. HTTP responses with `401` or `403` status codes are handled by interceptors though.